src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```
@@ -5,10 +5,29 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMUM_AGE = 18;
+    private static final int MINIMUM_LOGIN_LENGTH = 6;
+    private static final int MINIMUM_PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (storageDao.get(user.getLogin()) != null) {
+            throw new ValidationException("LOGIN " + user.getLogin() + "is used");
+        }
+        if (user.getPassword() != null) {
```

If password is null we need to throw an exception to let client know about it